UPDATING.md

```diff
@@ -24,6 +24,10 @@ assists users migrating to a new version.
 
 ## Airflow Master
 
+#### Remove run_duration
+
+We should not use the `run_duration` option anymore. This used to be for restarting the scheduler from time to time, but right now the scheduler is getting more stable and therefore using this setting is considered bad and might cause an inconsistent state.
+
```

Comment:
Is this a part of this PR? This is already a part of https://github.com/apache/incubator-airflow/pull/4320